examples/non-monorepo/README.md

```diff
@@ -0,0 +1,45 @@
+# Turborepo non-monorepo starter
+
+This is an official starter Turborepo.
+
+## What's inside?
+
+This Turborepo uses a single, non-monorepo project (in this case, a single Next.js application). Since [Turborepo 1.6](https://turbo.build/blog/turbo-1-6-0#any-codebase-can-use-turborepo), you can use Turborepo for non-monorepo projects as well as monorepos.
+
+## Using this example
+
+Run the following command:
+
+```sh
+npx degit vercel/turbo/examples/non-monorepo non-monorepo
+cd non-monorepo
+yarn install
+git init . && git add . && git commit -m "Init"
+```
+
+### Build
+
+To build all apps and packages, run the following command:
+
+```
+pnpm turbo build
+```
+
+### Develop
+
+To develop all apps and packages, run the following command:
+
+```
+pnpm turbo dev
+```
```

Comment:
Both of these should use `yarn`

Reply:
This should work with any package manager! I used pnpm for the README because it's generally our default

Reply:
I do prefer self-consistent, though. We use `yarn` in other places here.